ambry-api/src/test/java/com.github.ambry/rest/RestUtilsTest.java

```diff
@@ -127,7 +139,7 @@ public void getBlobPropertiesVariedInputTest() throws Exception {
     verifyBlobPropertiesConstructionFailure(headers, RestServiceErrorCode.InternalServerError);
 
     // no failures.
-    // ttl missing. Should be infinite time by default.
+    // ttl missing. Should be infinite time in non ttlRequired container.
```

Comment:
"should be" is misleading without the "by default" ("Should be infinite time by default in non ttlRequired container.")